### PR TITLE
release v-1.19.0

### PR DESCRIPTION
**Pull request by SaitejaTamma**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Part of #2381 

**Special notes for your reviewer**:

### CODE REVIEW

File: site/content/en/docs/Installation/Install Agones/helm.md [hunk 1]
```diff
@@ -94,8 +94,10 @@ The following tables lists the configurable parameters of the Agones chart and t
...
 | `agones.metrics.stackdriverLabels`                  | A set of default labels to add to all stackdriver metrics generated in form of key value pair (`key=value,key2=value2`). By default metadata are automatically added using Kubernetes API and GCP metadata enpoint.                              | \`\` |
+| `agones.metrics.serviceMonitor.interval`            | Default scraping interval for ServiceMonitor                                                    | `30s`                 |
 | `agones.serviceaccount.controller`                  | Service account name for the controller. **Note**: Will be replaced with `agones.serviceaccount.controller.name` in Agones 1.16 | `agones-controller`    |
-| `agones.serviceaccount.sdk`                         | Service account name for the sdk. **Note**: Will be replaced with `agones.serviceaccount.sdk.name` in Agones 1.16        | `agones-sdk`           |
+| `agones.serviceaccount.sdk`                         | Service account name for the sdk. **Note**: Will be replaced with `agones.serviceaccount.sdk.name` in Agones 1.16        | `agones-sdk`
```
Comment (roberthbailey) [inline site/content/en/docs/Installation/Install Agones/helm.md:99]: This line is missing the final `|` at the end.

File: site/config.toml [hunk 2]
```diff
@@ -82,9 +82,9 @@ github_repo = "https://github.com/googleforgames/agones"
...
 
 # current release branch. Never is rc.
-release_branch = "release-1.18.0"
+release_branch = "release-1.19.0"
 # the main version. Never is rc.
-release_version = "1.18.0"
+release_version = "1.19.0"
 
 # shown for production
 supported_k8s = "1.20"
```
Comment (roberthbailey) [inline site/config.toml:90]: The k8s versions should also be updated.

**Comment by roberthbailey** [comment]
Flake in sdk conformance tests:
```
make[2]: *** [includes/sdk.mk:145: run-sdk-conformance-no-build] Error 1
make[1]: *** [includes/sdk.mk:153: run-sdk-conformance-test] Error 2
make: *** [includes/sdk.mk:156: run-sdk-conformance-test-cpp] Error 2
```

**Comment by roberthbailey** [comment]
Even though the build failed I found the generated website at https://48f76ad-dot-preview-dot-agones-images.uc.r.appspot.com/site/ and verified that the changes look good.

**Review by roberthbailey — APPROVED** [review]

**Comment by roberthbailey** [comment]
feature-e2e tests timed out. Retrying again. :/

**Comment by roberthbailey** [comment]
cpp sdk conformance again:

```
make: *** [includes/sdk.mk:156: run-sdk-conformance-test-cpp] Error 2
```

This PR seems cursed.

**Comment by roberthbailey** [comment]
e2e-feature-gates (again):

```
--- FAIL: TestAutoscalerWebhook (378.40s)
    fleetautoscaler_test.go:523: 
        	Error Trace:	fleetautoscaler_test.go:523
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	TestAutoscalerWebhook
        	Messages:   	Received unexpected error
    fleetautoscaler_test.go:524: 
        	Error Trace:	fleetautoscaler_test.go:524
        	Error:      	Should be true
        	Test:       	TestAutoscalerWebhook
        	Messages:   	Expected error was not received
```

**Comment by roberthbailey** [comment]
I thought we'd done a good job reducing the flaky tests, but this PR is proving otherwise.

**Comment by roberthbailey** [comment]
e2e-feature-gates (again):

```
--- FAIL: TestFleetAutoScalerRollingUpdate (337.11s)
    fleetautoscaler_test.go:237: 
        	Error Trace:	fleetautoscaler_test.go:237
        	Error:      	Should be true
        	Test:       	TestFleetAutoScalerRollingUpdate
        	Messages:   	Could not get AbleToScale status
    fleetautoscaler_test.go:298: 
        	Error Trace:	fleetautoscaler_test.go:298
        	Error:      	Received unexpected error:
        	            	New replicas should be not less than (target - maxUnavailable)
        	            	agones.dev/agones/test/e2e.TestFleetAutoScalerRollingUpdate.func3
        	            		/go/src/agones.dev/agones/test/e2e/fleetautoscaler_test.go:285
        	            	k8s.io/apimachinery/pkg/util/wait.runConditionWithCrashProtection
        	            		/go/src/agones.dev/agones/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:211
        	            	k8s.io/apimachinery/pkg/util/wait.pollImmediateInternal
        	            		/go/src/agones.dev/agones/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:445
        	            	k8s.io/apimachinery/pkg/util/wait.PollImmediate
        	            		/go/src/agones.dev/agones/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:441
        	            	agones.dev/agones/test/e2e.TestFleetAutoScalerRollingUpdate
        	            		/go/src/agones.dev/agones/test/e2e/fleetautoscaler_test.go:275
        	            	testing.tRunner
        	            		/usr/local/go/src/testing/testing.go:1259
        	            	runtime.goexit
        	            		/usr/local/go/src/runtime/asm_amd64.s:1581
        	Test:       	TestFleetAutoScalerRollingUpdate
```

**Comment by roberthbailey** [comment]
e2e-feature-gates

```
--- FAIL: TestAutoscalerWebhook (375.34s)
    fleetautoscaler_test.go:523: 
        	Error Trace:	fleetautoscaler_test.go:523
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	TestAutoscalerWebhook
        	Messages:   	Received unexpected error
    fleetautoscaler_test.go:524: 
        	Error Trace:	fleetautoscaler_test.go:524
        	Error:      	Should be true
        	Test:       	TestAutoscalerWebhook
        	Messages:   	Expected error was not received
```

**Comment by markmandel** [comment]
There is definitely something flaky in the autoscaler/autoscaler tests - it pretty consistently fails there.

**Pull request merged**